src/pipe_mgr/shared/dal/dpdk/dal_tbl.c

```diff
@@ -54,12 +54,23 @@ static int table_match_field_info(char *table_name,
 			return BF_UNEXPECTED;
 		}
 
+		if (i == 0) {
+			max_offset = meta->mf[i].offset;
+			offset_min = max_offset;
+			add_bits = meta->mf[i].n_bits;
```

Comment:
may be just continue , if i ==0.

Reply:
yes, it should continue if i==0, and taken care